src/syncManager.ts

```diff
@@ -25,6 +26,7 @@ interface IParameters {
   resourceId: string;
   resourceVersion: string;
   layerRelativePath: string;
+  tocData?: JSON;
```

Comment:
use Record instead of JSON if it is object as it is more readable and the json interface values are always string.
if it is string use string instead